<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessarArquivoCrednosso</class>
 <widget class="QWidget" name="ProcessarArquivoCrednosso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Informações do arquivo</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QPushButton" name="botaoAtualizar">
        <property name="text">
         <string>Atualizar</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>396</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="botaoExportar">
        <property name="text">
         <string>Exportar</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="botaoConverter">
        <property name="text">
         <string>Converter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QLineEdit" name="campoLocalDoArquivo">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Caminho onde o arquivo (&lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; font-style:italic;&quot;&gt;TXT/DAT&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;) está localizado.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Endereço completo do arquivo selecionado...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="campoPesquisarObjetosTabela">
     <property name="placeholderText">
      <string>Pesquise um item na tabela abaixo digitando aqui...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>campoLocalDoArquivo</tabstop>
  <tabstop>botaoConverter</tabstop>
  <tabstop>botaoAtualizar</tabstop>
  <tabstop>botaoExportar</tabstop>
  <tabstop>campoPesquisarObjetosTabela</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
